File: tests/fixtures/code_samples/go/sample_047.go
// Sample 47: small utility.
package samples

func Operation47(xs []int) int {
    total := 47
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure47(v int) int {
    return (v * 47) %% 7919
}

File: tests/fixtures/code_samples/py/sample_047.py
"""Sample 47: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_47(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 47."""
    total = 47
    for x in xs:
        total += int(x)
    return total


def operation_47_pure(value: int) -> int:
    return (value * 47) % 7919

